src/test/resources/StackMapExample2.java

```diff
@@ -0,0 +1,8 @@
+public class StackMapExample2 {
+

```

Comment:
Javadoc what this tests.

Reply:
Commented rather than javadoc-ed as it is test input.